mlflow/projects/databricks.py

```diff
@@ -20,7 +20,8 @@
 from mlflow.exceptions import ExecutionException
 from mlflow.utils.mlflow_tags import MLFLOW_DATABRICKS_RUN_URL, MLFLOW_DATABRICKS_SHELL_JOB_ID, \
     MLFLOW_DATABRICKS_SHELL_JOB_RUN_ID, MLFLOW_DATABRICKS_WEBAPP_URL
-from mlflow.utils.uri import get_db_profile_from_uri, is_databricks_uri, is_http_uri
+from mlflow.utils.uri import get_db_info_from_uri, is_databricks_uri, is_http_uri,\
```

Comment:
`get_db_info_from_uri` is not used?

Reply:
I believe this was updated in one of the lint commits

Reply:
ah makes sense. i was confused why lint wasn't failing on it :-)